leetcode/math.md

```diff
@@ -0,0 +1,24 @@
+# Math
```

Comment:
VERIFICATION RESULT /nYour pull request LEETCODE-1022 Added base-7/n

Your commit: LEETCODE-1022 Added base-7

Your commit: LEETCODE 1022 Refactored base-7
You need to write "-" between prefix of your message and group number
Your commit: LEETCODE-1022 Refactored base-7

Your commit: Merge remote-tracking branch 'origin/base-7' into base-7

# Conflicts:
#	leetcode/math.md
You need to write "-" between prefix of your message and group number
Your commit: LEETCODE-1022 Refactored base-7

Your commit: LEETCODE-1022 Refactored base-7